hand-eye-calibration/full_hand_eye_sample.py

```diff
@@ -520,17 +479,23 @@ def perform_hand_eye_calibration(mode: str, data_dir: Path):
 
 def _main():
 
-    host = "192.168.1.246"
+    user_options = _options()
+
+    robot_IP_address = user_options.ip
+    # robot_IP_address = "192.168.1.246"
     port = 30004
-    con, input_data = _initialize_robot_sync(host, port)
+    con, input_data = _initialize_robot_sync(robot_IP_address, port)
```

Comment:
Will this throw an exception or return an error if the robot cannot be reached?
If so, I would suggest to handle error(s) and output an appropriate message.

Reply:
Yes, rtde handles connection and outputs an error if it is not able to connect.